pkg/netutil/netutil.go

```diff
@@ -121,49 +122,62 @@ func resolveURL(ctx context.Context, u url.URL) (string, error) {
 	return "", ctx.Err()
 }
 
+func sprintURLs(us []url.URL) []string {
```

Comment:
maybe change name to `urlsToStrings`? i think `sprintURLs` is specific to go's fmt naming.

Reply:
Fixed.